components/automate-ui/src/app/pages/notifications/notifications.component.html

```diff
@@ -16,10 +16,15 @@
 
       <chef-toolbar>
         <app-authorized [anyOf]="['/notifications/rules', 'post']">
-          <chef-button primary [routerLink]="['/settings/notifications/form']">Create Notification</chef-button>
+          <div *ngIf="(rules$ | async)?.length === 0" class="notifications-empty-state">
+            <p>Create the first notification to get started!</p>
+          </div>
+          <div [ngClass]="(rules$ | async)?.length === 0 ? 'notifications-empty-state' : ''">
+            <chef-button primary [routerLink]="['/settings/notifications/form']">Create Notification</chef-button>
+          </div>
         </app-authorized>
       </chef-toolbar>
-      <chef-table id="notifications-list">
+      <chef-table *ngIf="(rules$ | async)?.length > 0" id="notifications-list">
```

Comment:
Technically not a part of this task, but it would be very helpful to add the missing authorization here.
This `<chef-table>` should be wrapped in an `<app-authorized>` -- see the exact corollary in your other PR on tokens (https://github.com/chef/automate/pull/308/files).
The opening tag should be this:
```
<app-authorized [anyOf]="['/notifications/rules', 'get']">
```

Reply:
sure I will add, Thanks for the review and suggestions @msorens  :)

Reply:
I added changes, please have a look.